CONTRIBUTING.md

```diff
@@ -1,29 +1,39 @@
 # Guidelines for contributing
 
-Thank you for your interest in contributing to AWS documentation! We greatly value feedback and contributions from our community.
+Welcome to AWS Docs! We thrive on your contribution. :heart:
 
-Please read through this document before you submit any pull requests or issues. It will help us work together more effectively.
+## Before you get started
+Please take 3 minutes to understand:
+1. [Submit an issue](#submit-an-issue)
```

Comment:
I'd suggest changing this to 'What to contribute'

Reply:
That sort of misses the point of the section. These are 3 things I want the user to be aware of before startfing. Maybe I couldf addf tfhat as a sectifon?